corehq/blobs/__init__.py

```diff
@@ -34,4 +34,8 @@ def _get_fs_db(settings):
     return FilesystemBlobDB(blob_dir)
 
 
-BlobInfo = namedtuple("BlobInfo", ["name", "length", "digest"])
+class BlobInfo(namedtuple("BlobInfo", ["identifier", "length", "digest"])):
```

Comment:
Good idea to rename `name` to `identifier`. I had intended to rename it to `id` at some point, but hadn't gotten around to it yet.